src/Components/Web.JS/yarn.lock

```diff
@@ -346,16 +346,22 @@
     "@types/yargs" "^13.0.0"
 
 "@microsoft/dotnet-js-interop@link:../../JSInterop/Microsoft.JSInterop.JS/src":
-  version "0.0.0"
```

Comment:
@pranavkm do you know whether this one is correct?
I assume the `0.0.0` was intentionally left as such. Should I tough this?